### PR TITLE
Update to latest

### PR DESCRIPTION
**Pull request by jessfraz**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+golang-gocapability-dev (0.0~git20150324-1) UNRELEASED; urgency=low
+
+  * New upstream version (0.0~git20150324).
+
+ -- Johan Euphrosine <proppy@google.com>  Tue, 24 Mar 2015 13:36:10 -0700
```
Comment (tianon) [inline debian/changelog]: This should be "Jessie Frazelle" :wink:

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+golang-gocapability-dev (0.0~git20150324-1) UNRELEASED; urgency=low
+
+  * New upstream version (0.0~git20150324).
```
Comment (tianon) [inline debian/changelog:3]: For a second I thought this was a mistake, but nope, https://github.com/syndtr/gocapability/commit/8e4cdcb3c22b40d5e330ade0b68cb2e2a3cf6f98 (it really is latest commit today!)

Comment (jessfraz) [inline debian/changelog:3]: that was most def a coincidence as much as I would like to say i knew

File: debian/changelog (hunk 1 above)
Comment (jessfraz) [inline debian/changelog]: updated!

**Comment by tianon** [comment]
Oh crappers, I had already done this one in `debian-experimental` and forgotten. :angel:

**Pull request closed without merging**